front-end/src/components/PartnersBanner/index.tsx

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
 import { StyleSheet, css } from 'aphrodite';
 import themeVariables from '../../themeVariables.module.scss';
-import { STRINGS } from '../../strings';
+import {Row, Col} from 'react-bootstrap'
```

Comment:
Please add spacing around `{ Row, Col }` and a `;` at the end of the line. Let's also move this up since it's a package import, vs `themeVariables` which is local. It should go below the `aphrodite` import.